src/nowsecure-client.ts

```diff
@@ -8,9 +8,13 @@ import * as http from "@actions/http-client";
 import type {
   UploadApplicationResponse,
   PullReportResponse,
-} from "./nowsecure-types";
+} from "./types/platform";
```

Comment:
In case these imports are intentionally sorted alphabetically then this would need to be reordered to keep it that way.